modules/user-guide/pages/using-vale-github-action.adoc

```diff
@@ -1,33 +1,39 @@
 // Metadata for Antora
-:navtitle: GitHub Action
+:navtitle: Integrating Vale into your GitHub workflow
 :keywords: vale, github
-:description: Overview of using Vale in a GitHub Action
+:description: Integrating the Vale linter into your GitHub workflow by using a GitHub Action
 :page-aliases: end-user-guide:using-vale-github-action.adoc
 // End of metadata for Antora
 
 :context: using-vale-github-action
 :_module-type: PROCEDURE
 [id="proc_using-vale-github-action_{context}"]
-= Using Vale GitHub Action
+= Integrating Vale into your Git workflow by using a GitHub Action
 
-Consider using Vale GitHub Action when:
+Integrate Vale into your GitHub content development workflow by creating a GitHub Action that uses the errata-ai `vale-action`, which is the official GitHub Action for Vale.
 
-* The goal is to measure or enforce the project compliance with the Style Guides.
-* The content repository is on GitHub.
+By creating a GitHub Action workflow for vale, you can configure how vale runs including the Git workflow events that trigger the vale job, and the specific options to run vale with. For example, you can decide to:
 
-WARNING: Due to the GitHub token permissions, this Action can not post annotations to pull requests from forked repositories.
+* Run vale when a contributor creates a pull request
+* Linter the diff only
+* Allow the merging of pull requests if the report has errors
+* Block the merging of pull requests if the report has errors
 
-.Prerequisites
+For a full list of supported configuration options, see link:https://github.com/errata-ai/vale-action[Vale GitHub Action].
 
-* xref:installing-vale-cli.adoc[]
-* xref:adding-vale-configuration-to-a-project.adoc[]
-* xref:defining-a-vale-onboarding-strategy.adoc[]
+.Prerequisites
+* Your content repository is on GitHub. For GitLab repos, see * xref:defining-a-vale-onboarding-strategy.adoc[Define a Vale CI onboarding strategy for your project].
```

Comment:
```suggestion
* Your content repository is on GitHub. For GitLab repos, see xref:defining-a-vale-onboarding-strategy.adoc[Define a Vale CI onboarding strategy for your project].
```

Reply:
We can remove entirely "Lint the diff only". The GitHub action runs vale on all files and only reports back in the pull request errors in modified files.

Reply:
```
with:
   files: __onlyModified
   onlyAnnotateModifiedLines: true
```
The above config is now the default behavior in the ReviewDog implementation. You can set this to false, which is why I added it as a possibility to the list.